avocado/plugins/human.py

```diff
@@ -98,3 +99,22 @@ def post_tests(self, job):
                       job.result.errors, job.result.failed, job.result.skipped,
                       job.result.warned, job.result.interrupted)
         self.log.info("TESTS TIME : %.2f s", job.result.tests_total_time)
+
+
+class HumanJob(JobPre, JobPost):
+
+    """
+    Human result UI
+    """
+
+    name = 'human'
+    description = "Human Interface UI"
+
+    def pre(self, job):
+        pass
+
+    def post(self, job):
+        if job.time_elapsed != -1:
+            if not getattr(job.args, 'stdout_claimed_by', None):
+                log = logging.getLogger("avocado.app")
+                log.info("JOB TIME   : %.2f s", job.time_elapsed)
```

Comment:
OK, so the Human result plugin now becomes result and post tests plugin :smile:. A small step for avocado a big step for human plugin, spreading it's power.

^^ which means ack.